beacon-chain/core/helpers/beacon_committee.go

```diff
@@ -305,20 +305,15 @@ func CommitteeAssignments(ctx context.Context, state state.BeaconState, epoch pr
 	}
 	assignments := make(map[primitives.ValidatorIndex]*CommitteeAssignment)
 
-	activeValidatorCount, err := ActiveValidatorCount(ctx, state, epoch)
-	if err != nil {
-		return nil, err
-	}
-	ptcPerSlot, PtcMembersPerCommittee := PtcAllocation(activeValidatorCount)
-
 	// Compute committee assignments for each slot in the epoch.
 	for slot := startSlot; slot < startSlot+params.BeaconConfig().SlotsPerEpoch; slot++ {
 		committees, err := BeaconCommittees(ctx, state, slot)
 		if err != nil {
 			return nil, errors.Wrap(err, "could not compute beacon committees")
 		}
+		ptcPerSlot, ptcMembersPerCommittee := PtcAllocation(len(committees))
```

Comment:
Previously, we only needed to call `PtcAllocation` once, but now we need to call it 32 times. Given that the active validator count does not change throughout the epoch, we could just call it once. `PtcAllocation` is very cheap anyway, this is feedback for a micro-optimization and can be ignored...